FeedStoreChallenge/Infrastructure/CoreData/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Vladimir Mironiuk on 04.09.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
```

Comment:
`ManagedFeedImage` is not supposed to be subclassed, so it should be marked as `final`.
```suggestion
final class ManagedFeedImage: NSManagedObject {
```